lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -1064,20 +1132,6 @@ LogicalResult BuildOpGroups::buildOp(PatternRewriter &rewriter,
   return success();
 }
 
-LogicalResult BuildOpGroups::buildOp(PatternRewriter &rewriter,
```

Comment:
Trying to handle stages within BuildOpGroups was not going to work. It is not simple control flow that can be handled in a walk of the IR (at least not in the order BuildOpGroups is currently walking), so I made a separate lowering pattern for handling the stages _after_ BuildOpGroups.